src/CTA.Rules.Config/Utils.cs

```diff
@@ -15,6 +16,13 @@ namespace CTA.Rules.Config
     public class Utils
     {
         private const string DefaultMutexName = "DefaultMutex";
+        private static List<SolutionProjectType> AcceptedProjectTypes = new List<SolutionProjectType>()
```

Comment:
nit: Can we change `AcceptedProjectTypes` to a HashSet?

Reply:
also can do yup